modules/nw-dpu-configuring-operator.adoc

```diff
@@ -4,37 +4,47 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="nw-dpu-configuring-operator_{context}"]
-= Configuring the DPU Operator 
+= Configuring the DPU Operator
+
+[role="_abstract"]
+You can configure the DPU Operator after installation to enable management of DPU devices and network attachments in both dual cluster and single cluster deployment modes.
+
+You can configure the DPU Operator to manage the DPU devices and network attachments in your cluster.
 
 To configure the DPU Operator follow these steps:
 
 .Procedure
 
-. Create a `DpuOperatorConfig` custom resource (CR) on both the host cluster and on each of the DPU clusters. The DPU Operator in each cluster is activated after this CR is created.
+. Create the `DpuOperatorConfig` Custom Resource (CR) based on your deployment mode:
+
+* Dual Cluster Deployment: You must create the `DpuOperatorConfig` CR on both the host {product-title} cluster and on each of the {ms} DPU clusters.
+* Single Cluster Deployment: This deployment uses a standard {product-title} cluster. You only need to create the `DpuOperatorConfig` CR once on this cluster.
++
+The content of the CR is the same for all clusters.
 
-. Create a file named `dpu-operator-host-config.yaml` by using the following YAML:
+. Create a file named `dpu-operator-config.yaml` by using the following YAML:
 +
 [source,yaml]
 ----
 apiVersion: config.openshift.io/v1
 kind: DpuOperatorConfig
 metadata:
- name: dpu-operator-config <1>
+ name: dpu-operator-config
 spec:
- mode: host <2>
+ logLevel: 0
 ----
 +
-<1> The name of the custom resource must be `dpu-operator-config`.
-<2> Set the value to `host` on the host cluster. On each DPU cluster, which runs a single MicroShift cluster per DPU, set the value to `dpu`.
+* `metadata.name`: Specifies the name of the Custom Resource, which must be `dpu-operator-config`.
+* `spec.logLevel`: Sets the desired logging verbosity in the operator container logs. The value `0` is the default setting.
 
 . Create the resource by running the following command:
 +
 [source,terminal]
 ----
-$ oc apply -f dpu-operator-host-config.yaml
+$ oc apply -f dpu-operator-config.yaml
 ----
 
-. You must label all nodes that either have an attached DPU or are functioning as a DPU. On the host cluster, this means labeling all compute nodes assuming each node has an attached DPU with `dpu=true`. On the DPU, where each MicroShift cluster consists of a single node, label that single node in each cluster with `dpu=true`. You can apply this label by running the following command:
+. You must label all nodes that either have an attached DPU or are functioning as a DPU. You can apply this label by running the following command:
```

Comment:
this is strange wording "You must ...."
It's kinda implied in the procedure that you must do all the steps unless they're marked as optional.

```suggestion
. Label all nodes that either have an attached DPU or are functioning as a DPU. You can apply this label by running the following command:
```